Version.xcconfig

```diff
@@ -7,8 +7,8 @@
 //
 
 // Version [DEFAULT]
-LOOP_MARKETING_VERSION = 2.3.0
-CURRENT_PROJECT_VERSION = 57
+LOOP_MARKETING_VERSION = 0.0.1
```

Comment:
This will need approval from @ps2. I think when this is merged back to DIY it is going to cause all sorts of confusion.

Reply:
Yes, I think the default here should remain the DIY version, and we should continue using our overridden version. I also think keeping the CURRENT_PROJECT_VERSION is useful for those building for testflight to have a single place to change it easily.

Reply:
Right.  Ok I will revert this.